combat.py

```diff
@@ -0,0 +1,88 @@
+from adventurelib import *
+
+#by default, cannot fight
+Room.can_attack = False
+#create dungeon room
+dungeon = Room("a dark, scary dungeon under the castle")
+dungeon.can_attack = True
+#set current room to dungeon
+current_room = dungeon
+
+#create enemies in dungeon
+dungeon.enemies = Bag()
+v1 = Item("Vampire")
+v1.weakness = "Garlic"
+v2 = Item("Shrek")
+v2.weakness = "Onion"
+dungeon.enemies.add(v1)
+
+#create "hand" to represent equipped item
+hand = Bag()
+
+#temporary inventory for testing
+inv = Bag()
+inv.add(Item("Garlic"))
+inv.add(Item("Onion"))
+
```

Comment:
Everything above here needs to be encapsulated in its own file

Reply:
Now encapsulated in swamp.py